estate/models/estate_property.py

```diff
@@ -0,0 +1,42 @@
+from odoo import fields, models
+from datetime import timedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate property"
+
+    name = fields.Char('Name', required=True, default='My new house')
+    description = fields.Text('Description')
+    active = fields.Boolean(default=True)
```

Comment:
```suggestion
```
The `active` field is a special field in Odoo that is automatically created on every model and is `True` by default. You do not need to declare it unless you want to change its default behavior.